packages/core/fs/src/NodeFS.js

```diff
@@ -222,11 +224,43 @@ try {
   //
 }
 
+let useOsTmpDir;
+function shouldUseOsTmpDir(filePath) {
+  if (useOsTmpDir != null) {
+    return useOsTmpDir;
```

Comment:
I guess there's a tiny risk the `/tmp` directory becomes un-usable between the first check and the first write, but that seems pretty unlikely?

Reply:
I can't even imagine how that would happen. I'm comfortable with a single check.